model-optimizer/extensions/front/CTCGreedyDecoderReplacement.py

```diff
@@ -1,12 +1,15 @@
 # Copyright (C) 2018-2021 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
 
+import logging as log
+
 from extensions.ops.ctc_greedy_decoder_seq_len import CTCGreedyDecoderSeqLenOp
 from extensions.ops.transpose import Transpose
 from mo.front.common.partial_infer.utils import int64_array
 from mo.front.common.replacement import FrontReplacementSubgraph
 from mo.front.tf.graph_utils import create_op_with_const_inputs
 from mo.graph.graph import Graph, rename_nodes
+from mo.ops.result import Result
 
```

Comment:
also, I think it makes sense to move this file to `front/tf` because these transformations are TF specific. For ONNX models they are not valid

Reply:
Yes, I agree. File `CTCGreedyDecoderReplacement.py` and `CTCLossReplacement.py` and appropriate tests were replaced to `front/tf` folder from `front`.